quinn/examples/common/mod.rs

```diff
@@ -0,0 +1,77 @@
+//! Commonly used code in most examples.
+
+use quinn::{
+    Certificate, CertificateChain, ClientConfig, ClientConfigBuilder, Driver, Endpoint, Incoming,
+    PrivateKey, ServerConfig, ServerConfigBuilder, TransportConfig,
+};
+use std::error::Error;
+use std::net::ToSocketAddrs;
+use std::sync::Arc;
+
+/// Returns QUIC endpoint builder configured to build a client endpoint.
+///
+/// ## Args
+///
+/// - server_certs: list of trusted certificates.
+#[allow(unused)]
```

Comment:
Why is this `allow` needed?

Reply:
`insecure_connection` example does not use this function and when it includes `common` module the compiler complains about unused function.